### ISSUE TITLE
Meet vs Free

### ISSUE THREAD

**Issue by MirjamMaahs** [CindyJS/CindyJS#288]
In the following example, there are two canvas windows. The right one should be the duality from the left one. 
So point A should be the intersection of the three points on line from the first window. If the type of A (in object cdy2) is "Meet", it doesn't work. In contrast if the type is "Free".
How is this explainable?

```html
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>PrimalDual.cdy</title>
    <style type="text/css">
        * {
            margin: 0px;
            padding: 0px;
        }

        #CSConsole {
            background-color: #FAFAFA;
            border-top: 1px solid #333333;
            bottom: 0px;
            height: 200px;
            overflow-y: scroll;
            position: fixed;
            width: 100%;
        }
    </style>
    <script type="text/javascript" src="http://cinderella.de/CindyJS/latest/Cindy.js"></script>
    <script id='csdraw' type='text/x-cindyscript'>
    javascript("cdy2.evokeCS('a.homog=" + A.homog + "; b.homog=" + B.homog + "; A.homog = cross(a.homog, b.homog); c.homog=" + C.homog + "; d.homog=" + D.homog + ";')");
    </script>

    <script type="text/javascript">
        var cdy2 = createCindy({
	defaultAppearance: { fontFamily: "sans-serif", lineSize: 1, pointSize: 5.0 }, 
	angleUnit: "°", 
	geometry: [  
		{ name: "a", type: "FreeLine", pos: [ 4.0, -1.3124999999999998, -1.0050752737226278 ], color: [ 1.0, 0.0, 0.0 ], labeled: true },
        { name: "b", type: "FreeLine", pos: [ 1.9009900990099013, -4.0, 0.636879381368794 ], color: [ 1.0, 0.0, 0.0 ], labeled: true },
        { name: "c", type: "FreeLine", pos: [ 4.0, 3.822222222222223, 0.714720194647202 ], color: [ 1.0, 0.0, 0.0 ], labeled: true },
        { name: "A", type: "Meet", color: [ 0.0, 0.0, 1.0 ], args: [ "a", "b" ], labeled: true },
        { name: "d", type: "Through", pos: [ 4.0, -2.6269385437590445, -0.5711624172242649 ], color: [ 0.098039225, 0.61960787, 0.30588236 ], args: ["A"], labeled: true }
        ],
	ports: [ 
		{ id: "CSCanvas2", width: 420, height: 355, transform: [ { visibleRect: [ -9.06, 9.34, 18.14, -4.1 ] } ], background: "rgb(168,176,192)" } ], 
	cinderella: { build: 1801, version: [ 2, 9, 1801 ] } });
var cdy1 = createCindy({ 
	scripts: "cs*", 
	defaultAppearance: { fontFamily: "sans-serif", lineSize: 1, pointSize: 5.0 }, 
	angleUnit: "°", 
	geometry: [ 
		{ name: "A", type: "Free", pos: [ 4.0, -1.3124999999999998, -1.0050752737226278 ], color: [ 1.0, 0.0, 0.0 ], labeled: true }, 
		{ name: "B", type: "Free", pos: [ 1.9009900990099013, -4.0, 0.636879381368794 ], color: [ 1.0, 0.0, 0.0 ], labeled: true }, 
		{ name: "a", type: "Join", color: [ 0.0, 0.0, 1.0 ], args: [ "A", "B" ], labeled: true }, 
		{ name: "C", type: "Free", pos: [ 4.0, 3.822222222222223, 0.714720194647202 ], color: [ 1.0, 0.0, 0.0 ], labeled: true }, 
		{ name: "D", type: "PointOnLine", pos: [ 4.0, -2.6269385437590445, -0.5711624172242649 ], color: [ 0.098039225, 0.61960787, 0.30588236 ], args: [ "a" ], labeled: true } ], 
	ports: [ 
		{ id: "CSCanvas1", width: 423, height: 355, transform: [ { visibleRect: [ -9.06, 9.34, 18.14, -4.1 ] } ], background: "rgb(168,176,192)" }],  
	cinderella: { build: 1801, version: [ 2, 9, 1801 ] } });
        
    </script>
</head>
<body>
    <canvas id="CSCanvas1"></canvas>
    <canvas id="CSCanvas2"></canvas>
</body>
</html>
```

**Comment by gagern** [comment]
This is due to #283 and should be fixed by the #287 I just merged.

**Comment by kranich** [comment]
@MirjamMaahs Thanks for reporting this.
